### ISSUE TITLE
PDF loads perfectly, works in both orientations on iOS, but not on Android

### ISSUE THREAD

**Issue by utrayn** [lubritto/pdf_viewer_plugin#33]
Hey! First of all, thank you for developing this plugin. 

I use it to load a remote PDF from a URL. Everything works perfectly fine on iOS. 

Even on Android PDF loads in one orientation. However, on orientation change (from landscape to portrait or from portrait to landscape), it throws a white blank screen. Rotating in any orientation on Android gives a blank screen. Initially on page load, everything works fine. But as soon as the orientation changes, white blank screen.

**Comment by jglatre** [comment]
I've found a workaround. Basically you have to listen to window size changes and rebuild PdfViewer for every change.

Do the following:

1. Use mixin `WidgetsBindingObserver` and implement method `didChangeMetrics()`
2. In that method feed a StreamController or rxdart Subject with `WidgetsBinding.instance.window.physicalSize`
3. Wrap your PdfViewer with a StreamBuilder, something like this:
```
StreamBuilder<Size>(
  stream: referenceToStreamControllerOrSubject,
  initialData: Size.zero,
  builder: (_, snapshot) => PdfViewer(
    key: ObjectKey(snapshot.data.toString()),
    filePath: someFilePath
  )
)
```
Works in Android, iOS still not checked.

**Comment by jglatre** [comment]
There's a much simpler way to achieve the same effect:
```
OrientationBuilder(
  builder: (_, orientation) => PdfViewer(
    key: ObjectKey(orientation),
    filePath: someFilePath
  )
)
```

**Comment by lubritto** [comment]
This issue was fixed on version 2.0.0
